docs/editor/settings-sync.md

```diff
@@ -124,30 +124,31 @@ If you want to remove all your data from our servers, just turn off sync via **S
 
 ### Is VS Code Settings Sync the same as the Settings Sync extension?
 
-No, the [Settings Sync](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) extension by [Shan Khan](https://marketplace.visualstudio.com/publishers/Shan) uses a private Gist on GitHub to share your VS Code settings across different machines and is unrelated to the VS Code Settings Sync.
+No, the [Settings Sync](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) extension by [Shan Khan](https://marketplace.visualstudio.com/publishers/Shan) uses a secret GitHub Gist to coordinate your private VS Code configuration settings among connected machines, which is unrelated to the VS Code 'Settings Sync' extension.
```

Comment:
Current text is fine and more straightforward. 
Settings Sync is not an extension, it is a built-in feature.